packages/aws-cdk-lib/rosetta/aws_appsync/default.ts-fixture

```diff
@@ -1,11 +1,12 @@
 // Fixture with packages imported, but nothing else
-import { RemovalPolicy, Stack } from 'aws-cdk-lib';
+import { Duration, RemovalPolicy, Stack, CfnOutput } from 'aws-cdk-lib';
 import { Construct } from 'constructs';
 import appsync = require('aws-cdk-lib/aws-appsync');
 import ec2 = require('aws-cdk-lib/aws-ec2');
 import dynamodb = require('aws-cdk-lib/aws-dynamodb');
 import iam = require('aws-cdk-lib/aws-iam');
 import rds = require('aws-cdk-lib/aws-rds');
+import * as wafv2 from 'aws-cdk-lib/aws-wafv2';
```

Comment:
These are the assumed import statements for examples in the documentation. When you run the rosetta script, if you don't have a specific ts-fixture file defined, it uses a default file which has generic imports. Without this file, running rosetta would fail for the examples defined in the AWS AppSync README. More info here in the [Contributing guide](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md#documentation).